src/coguard_cli/discovery/config_file_finders/config_file_finder_helm.py

```diff
@@ -0,0 +1,163 @@
+"""
+This module contains the class to find Helm configurations
+inside a folder structure.
+"""
+
+import os
+import re
+import tempfile
+import logging
+from typing import Dict, List, Optional, Tuple
+import yaml
+from flatten_dict import unflatten
+from coguard_cli.discovery.config_file_finder_abc import ConfigFileFinder
+from coguard_cli.print_colors import COLOR_CYAN, COLOR_TERMINATION
+from coguard_cli.image_check import docker_dao
```

Comment:
Move that maybe outside of the image_check, since this is now used on both ends.